addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1513,7 +1513,7 @@ export class OdooEditor extends EventTarget {
             doJoin &&
             next &&
             !(next.previousSibling && next.previousSibling === joinWith) &&
-            this.editable.contains(next)
+            this.editable.contains(next) && !isNotEditableNode(next)
```

Comment:
I tried running the test without the change in this line and it seemed to work correctly. Could you please give it a try and let me know ? Thanks !

Reply:
@dmo-odoo, The testing team has mentioned two reviews in the pad.For point number two i have to add this line and the first point has to be solved.